src/open_inwoner/templates/pages/profile/contacts/list.html

```diff
@@ -22,11 +61,13 @@ <h1 class="h1" id="title">
         </tr>
     </thead>
     <tbody>
-        {% if not pending_invitations and not pending_approvals and not object_list %}
+        {% if not pending_invitations and not contacts_for_approval and not pending_approval and not pending_approval_list and not object_list %}
```

Comment:
(this is a lot.. but then I don't know how to do it differently without making it worse so meh :wink: )